README.md

```diff
@@ -321,6 +321,7 @@ Native desktop applications for managing and montoring docker hosts and clusters
 - [dockercraft](https://github.com/docker/dockercraft) - Docker + Minecraft = Dockercraft by [@docker][docker]
 - [dockerfile-mode](https://github.com/spotify/dockerfile-mode) An emacs mode for handling Dockerfiles by [@spotify][spotify]
 - [dockersql](https://github.com/crosbymichael/dockersql) - A command line interface to query Docker using SQL by [@crosbymichael](https://github.com/crosbymichael)
+- [DockSTARTer](https://dockstarter.com/) - DockSTARTer helps you get started with home server apps running in Docker by  ([GhostWriters](https://github.com/GhostWriters))
```

Comment:
Please, replace the website with the github project URL: https://github.com/GhostWriters/DockSTARTer

Reply:
No problem, I've added another commit to this PR to use the GitHub URL.